b/boringssl/boringssl_ubuntu_18.04.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : boringssl
+# Version       : n/a
```

Comment:
Please use the commit id (36a41bf), if you wanted to build and validate the master branch.
This keep our script consistent.